rotkehlchen/chain/ethereum/modules/lido_csm/balances.py

```diff
@@ -0,0 +1,117 @@
+import logging
+from collections import defaultdict
+from typing import TYPE_CHECKING
+
+from rotkehlchen.accounting.structures.balance import Balance, BalanceSheet
+from rotkehlchen.assets.utils import asset_normalized_value
+from rotkehlchen.chain.ethereum.interfaces.balances import BalancesSheetType, ProtocolWithBalance
+from rotkehlchen.chain.ethereum.modules.lido_csm.constants import (
+    ACCOUNTING_ABI,
+    CPT_LIDO_CSM,
+    LIDO_CSM_ACCOUNTING_CONTRACT,
+    STETH_ABI,
+)
+from rotkehlchen.chain.ethereum.modules.lido_csm.metrics import LidoCsmMetricsFetcher
+from rotkehlchen.chain.evm.contracts import EvmContract
+from rotkehlchen.constants.assets import A_STETH
+from rotkehlchen.constants.misc import ZERO
+from rotkehlchen.db.lido_csm import DBLidoCsm
+from rotkehlchen.errors.misc import RemoteError
+from rotkehlchen.fval import FVal
+from rotkehlchen.inquirer import Inquirer
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.ethereum.decoding.decoder import EthereumTransactionDecoder
+    from rotkehlchen.chain.ethereum.node_inquirer import EthereumInquirer
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+class LidoCsmBalances(ProtocolWithBalance):
+    """Queries Lido CSM accounting contract for node operator bond balances."""
+
+    def __init__(
+            self,
+            evm_inquirer: 'EthereumInquirer',
+            tx_decoder: 'EthereumTransactionDecoder',
+    ) -> None:
+        super().__init__(
+            evm_inquirer=evm_inquirer,
+            tx_decoder=tx_decoder,
+            counterparty=CPT_LIDO_CSM,
+            deposit_event_types=set(),
+        )
+        self._node_operator_db = DBLidoCsm(self.event_db.db)
+        self._steth_token = A_STETH.resolve_to_evm_token()
```

Comment:
This may be overoptimization here but you can probably skip saving the steth_token. You just need the steth address right? Just take it and use it all around here in this class instead of saving an asset in memory of the class and rresolving it to evm token.

For the `asset_normailzed_value` function you can just use the one that takes decimals directly and use the default value of 18 as we know the value of the token and this won't ever change.